matrix/matrix.py

```diff
@@ -394,6 +396,10 @@ def skydive_output(matrix):
 
 def main():
 
+    if environ.get('DISPLAY') is None:
+        print("skydive-flow-matrix need run on pc with GUI")
```

Comment:
is it still true is we choose another engine or output format ?

Reply:
Hmm. may be relocate check when run render?

Reply:
yes sounds good, thx

Reply:
relocated